stdlib/task/src/main/java/org/ballerinalang/stdlib/task/utils/TaskExecutor.java

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ *
+ */
+package org.ballerinalang.stdlib.task.utils;
+
+import org.ballerinalang.bre.Context;
+import org.ballerinalang.bre.bvm.BVMExecutor;
+import org.ballerinalang.model.values.BValue;
+import org.ballerinalang.stdlib.task.objects.ServiceWithParameters;
+import org.ballerinalang.util.codegen.FunctionInfo;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * This class invokes the Ballerina onTrigger function, and if an error occurs while invoking that function, it invokes
+ * the onError function.
+ */
+public class TaskExecutor {
+
+    public static void execute(Context context, ServiceWithParameters serviceWithParameters) {
```

Comment:
Why do we need to pass the context here?

Reply:
Context is not needed. Removed it.